htseq.wdl

```diff
@@ -0,0 +1,25 @@
+task HTSeqCount {
+    Array[File] alignmentFiles
+    File gffFile
+
+    String? format
+    String? order
+    String? stranded
+
+    String? preCommand
+
+    command {
+        set -e -o pipefail
+        ${preCommand}
+        htseq-count \
+        -f ${default="bam" format} \
+        -r ${default="pos" order} \
+        -s ${default="no" stranded} \
+        ${sep=" " alignmentFiles} \
+        ${gffFile}
```

Comment:
For better control of the output file maybe ` > ${outputGffFile}` is better here. This because the stdout/stderr always endup in the execution directory